packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -1,85 +1,431 @@
 // Web standard events
+
+/**
+ * Fired when the grid is resized. Called with a [[GridResizeParams]] object.
+ * @event
+ */
 export const GRID_RESIZE = 'resize';
+
+/**
+ * Fired when a `focusout` event happens in the grid.
+ * @event
+ */
 export const GRID_FOCUS_OUT = 'focusout';
+
+/**
+ * Fired when a `keydown` event happens in the grid.
+ * @event
+ */
 export const GRID_KEYDOWN = 'keydown';
+
+/**
+ * Fired when a `keyup` event happens in the grid.
+ * @event
+ */
 export const GRID_KEYUP = 'keyup';
+
+/**
+ * @ignore - do not document.
+ */
 export const GRID_NATIVE_SCROLL = 'scroll';
-export const GRID_DRAGEND = 'dragend';
 
 // GRID events
+
+/**
+ * Fired when an exception is thrown in the grid.
+ * @event
+ */
 export const GRID_COMPONENT_ERROR = 'componentError';
+
+/**
+ * Fired when the grid is unmounted.
+ * @event
+ */
 export const GRID_UNMOUNT = 'unmount';
+
+/**
+ * @ignore - do not document.
+ */
 export const GRID_ELEMENT_FOCUS_OUT = 'gridFocusOut';
```

Comment:
I ignored it because I think this event could be removed. It's only used to detect if the multiple key is pressed and we agreed that the `event` argument should be used a single source of truth. Also, its name is confusing and hard to explain the conditions in which the event is fired.